### DISCUSSION TITLE
App missing a code signature

### DISCUSSION THREAD

**Discussion by hairybridget** [FreeTubeApp/FreeTube#5260] [Q&A]
Looks like the dmg package is missing a code signature:

Unsigned ('errSecCSUnsigned')

Is there a signed version?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
There is no signed version

> The problem is that Apple (and Microsoft) expects every application to be "Signed" with a verified certificate that tells the user that the software that you are about to install comes from the verified source that it claims to be. The issue with getting FreeTube signed is that it involves paying Apple a yearly subscription to their developer program (Which is about ~$100 IIRC) just for the privilege of having their software signed (Microsoft charges even more).

Also see https://docs.freetubeapp.io/faq/#my-virus-protection-gave-a-warning-when-i-tried-to-run-freetube-is-freetube-a-malicious-application
